hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -50,19 +50,31 @@
       tests: ["i2c_error_intr"]
     }
     {
-      name: stress_all_with_reset
-      desc: '''
-            Pull reset at random times, ensure DUT are reset and recovered/ correctly and
-            there is no residual data left in the registers.
+      name: stress_all
+      desc: ''',
+            Support vseq (context) switching with random reset in between.
 
             Stimulus:
-              - Randomly reset DUT
-              - Restart send/receive transaction
+              - Combine above sequences in one test to run sequentially
+                except csr sequence and i2c_rx_oversample_vseq (requires zero_delays)
+              - Randomly add reset between each sequence,
 
             Checking:
-              - Ensure registers are properly reset
+              - Ensure transactions are transceivered correctly
             '''
       milestone: V2
+      tests: ["i2c_stress_all"]
+    }
+    {
+      name: stress_all_with_rand_reset
+      desc: '''
+            Support random reset in parallel with stress_all and tl_errors sequences.
+
+            Checking:
+              - Ensure transactions are transceivered correctly
+            '''
+      // TODO: temporally set to V3 to excluding from running regression test
+      milestone: V3
```

Comment:
If I understand correctly, this patch adds two tests: one works, the other is broken. Maybe split up the patch just to add the one that works, then fix the broken one before you push it up?

Reply:
@rswarbrick IMO, the testplan.md should include all the required tests regardless of DV/RTL development progress. We use *_sim_cfg.hjson to control which tests are run in regression and the stress_all_with_rand_reset is currently exclude. I am considering putting the milestone of `stress_all_with_rand_reset ` as V2. Again, this PR as shown in the title just focuses on the stress_all test.

Reply:
Sorry, I got confused by the re-ordering in `i2c_testplan.hjson`, which makes it look like you added new entries when in fact it's just diff not being able to say "lines A and B were swapped".